enlighten/timing/BatchCollection.py

```diff
@@ -345,13 +348,10 @@ def start_collection(self):
 
             # initialize driver-level laser auto_triggering and dark collection
             if self.laser_mode == "spectrum":
-                take_one_request = TakeOneRequest(take_dark=self.dark_before_batch, enable_laser_before=True, disable_laser_after=True, laser_warmup_ms=self.laser_warmup_ms)
+                self.take_one_template = TakeOneRequest(take_dark=self.dark_before_batch, enable_laser_before=True, disable_laser_after=True, laser_warmup_ms=self.laser_warmup_ms)
             else:
-                take_one_request = TakeOneRequest()
+                self.take_one_template = TakeOneRequest()
```

Comment:
This fixed the issue with "spectrum" laser_mode no longer working. Before, BatchCollection had been "pushing" a TakeOneRequest with the correct enable_laser_before and disable_laser_after fields in start_collection, but it was assuming that TakeOneRequest would then "persist" and be used throughout the batch. 

That's not how it works. BatchCollection works by virtually clicking the "Step-and-Save" (⏭️) button of VCRControls at timed intervals. VCRControls.step_and_save in turn calls TakeOneFeature.start, which generates its own TakeOneRequest to feed downstream into Wasatch.PY.

(There was a time when BatchCollection would "persist" settings like "auto_laser_enable" inside Wasatch.PY, but it got ugly and didn't scale, so we recently added TakeOneRequest so that all requested settings for a given measurement would be encapsulated in a single object, with a unique ID, so we could then "catch" the resulting spectrum and associate it, by ID, with the originating request.)

So anyway, my solution was for BatchCollection to define a "template" of how it wanted TakeOneRequests to be generated for the purpose of the batch, and then pass that template into VCRControls when triggering measurements.